plugins/callback/foreman.py

```diff
@@ -281,17 +334,40 @@ def send_reports(self, stats):
                 report['config_report']['status']['pending'] = total['changed']
                 report['config_report']['status']['applied'] = 0
 
-            self._send_data('report', host, report)
-
+            self._send_data('report', 'foreman', host, report)
             self.items[host] = []
 
+    def send_reports(self, stats):
+        if self.report_type == "foreman":
+            self.send_reports_foreman_legacy(stats)
+        elif self.report_type == "proxy":
+            self.send_reports_proxy_host_report(stats)
```

Comment:
why is the one called `foreman_legacy` and the other `proxy_host_report`? wouldn't `foreman` and `proxy` be sufficient? or will there also be a new foreman type in the future?

certainly not blocking, but wondering why the methods aren't just called `send_reports_<report_type>`, which would make reading the code easier

Reply:
Sure, yeah that is better.